File: sponge-examples/examples/standalone/error_reporting_trigger/error_reporting_trigger.py
"""
Sponge Knowledge base
Error reporting
"""

class HelloWorldTrigger(Trigger):
    def configure(self):
        self.eventName = "helloEvent"
    def run(self, event):
        whatIsThis.doSomething()
        print event.get("say")

def onStartup():
    EPS.event("helloEvent").set("say", "Hello World!").send()

File: sponge-examples/examples/standalone/error_reporting_trigger/error_reporting_trigger.js
/**
 * Sponge Knowledge base 
 * Error reporting
 */

var HelloWorldTrigger = Java.extend(Trigger, {
    configure: function(self) {
        self.eventName = "helloEvent";
    },
    run : function(self, event) {
        whatIsThis.doSomething();
        print(event.get("say"));
    }
});

function onStartup() {
    EPS.event("helloEvent").set("say", "Hello World!").send();
}
